### PR TITLE
chore(logging): add remote address chain to fxa content server

### PR DESCRIPTION
**Pull request by udaraweerasinghege**
Fixes https://github.com/mozilla/fxa-content-server/issues/5149

### CODE REVIEW

File: server/lib/logging/route_logging.js [hunk 1]
```diff
@@ -19,17 +19,31 @@ const disabled = function (req, res, next) {
...
 
-const formats = {
-  'default_fxa': (tokens, req, res) => JSON.stringify({
+function defaultFxaFormat(tokens, req, res) {
+  'use strict';
+  let xff = (req.headers['x-forwarded-for'] || '').split(/\s*,\s*/);
+  xff.push(req.ip || req.connection.remoteAddress);
+  // Remove empty items from the list, in case of badly-formed header.
+  xff = xff.filter(x => x);
+  let clientAddressIndex = xff.length - (config.get('clientAddressDepth') || 1);
```
Comment (vladikoff) [inline server/lib/logging/route_logging.js]: Not sure about other servers, but let's fix this here. Can we move the `config.get` out of the `defaultFxaFormat` function. Assign it to a variable after `const config = require('../configuration');` above.
This way we don't have to do the `.get` lookup all the time.

**Comment by udaraweerasinghege** [comment]
@vladikoff fixed, merge whenever you like :)

**Comment by vladikoff** [comment]
@udaraweerasinghege I see

```
FAIL: routeLogging - it logs a string if log format is dev_fxa (24ms)
TypeError: Cannot read property 'args' of null
  at Test.it logs a string if log format is dev_fxa [as test]  <tests/server/logging/route_logging.js:65:45>
  at <node_modules/intern/lib/Test.js:191:24>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/browser_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/browser_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:489:9>
  at process._tickCallback  <node.js:418:13>
FAIL: routeLogging - it logs a json blob if log format is not dev_fxa (2ms)
TypeError: Cannot read property 'args' of null
  at Test.it logs a json blob if log format is not dev_fxa [as test]  <tests/server/logging/route_logging.js:80:45>
  at <node_modules/intern/lib/Test.js:191:24>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/browser_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/browser_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:489:9>
  at process._tickCallback  <node.js:418:13>
2/2 tests failed
2/639 tests failed
```

in Travis. Do we get that locally?

**Comment by udaraweerasinghege** [comment]
@vladikoff my bad, i didnt update tests after changing config.  Updated tests now.

**Comment by vladikoff** [comment]
@udaraweerasinghege woot, thanks!

**Pull request merged**